filebeat/module/elasticsearch/server/ingest/pipeline.json

```diff
@@ -22,29 +28,8 @@
         },
         {
             "rename": {
-                "field": "@timestamp",
-                "target_field": "read_timestamp"
-            }
-        },
-        {
-            "date": {
                 "field": "timestamp",
-                "target_field": "@timestamp",
-                "formats": [
-                    "ISO8601",
-                    "YYYY-MM-dd HH:mm:ss,SSS"
-                ]
-            }
-        },
-        {
-            "remove": {
-                "field": "timestamp"
-            }
-        },
-        {
-            "append": {
-                "field": "service.name",
-                "value": "elasticsearch"
+                "target_field": "@timestamp"
```

Comment:
@tsg @ph This is needed because it seems inside grok I can't use `@timestamp`. Sound like the `@` symbol has it's issue.

Reply:
Yeah, I think you have a limited set of characters for grok keys.
Also, I haven't tested it personally but usually the extracted date isn't a 1:1 match, maybe this is why I never trip into it. :)

Also, did you get an error or you were not able to access the key?

Reply:
I did get an error from Elasticsearch, something like key not found.